openmetadata-ui/src/main/resources/ui/src/components/DataContract/ContractDetailTab/ContractDetail.tsx

```diff
@@ -457,10 +454,9 @@ const ContractDetail: React.FC<{
                 <Divider className="contract-dash-separator" />
               </div>
 
-              <RichTextEditorPreviewerNew
+              <RichTextEditorPreviewerV1
                 enableSeeMoreVariant
```

Comment:
What will happen if it increases by more than 3 lines? Have we tested it?

Reply:
Added SS above!